terraform/deployments/103495720024_exec_roles/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "environment" {
+  type        = string
+  description = "One of [ production | staging | demo ]"
+  default     = "staging"
+}
+
+variable "splunk_config_bucket" {
+  description = "Name of the target bucket for retriving config."
```

Comment:
very nitpicky but there's a typo in retrieving

Reply:
Thank you - I hate typos.